### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by rolandpeelen**
###  :notebook_with_decorative_cover: Description

This PR adds tabs to our component list.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions
Check the movie below, or try it out locally...

https://user-images.githubusercontent.com/4375473/131346414-a6c59e9a-87c3-4e9b-a90c-7f037cf3d32f.mov

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,8 +11,14 @@ Every entry has a category for which we use the following visual abbreviations:
 - ⚡️ Breaking Change
 - 🐞 Bugfix
 
+## [0.12.0]
+- 🎁 Tabs [#81](https://github.com/tenzir/ui-component-library/pull/81)
+We now have a tabs, and tabbed card component. They have built-in support for
+drag 'n drop replacing, updating, adding, duplicating and removing. Depending
```
Comment (dominiklohmann) [inline CHANGELOG.md:17]:
```suggestion
drag'n'drop replacing, updating, adding, duplicating and removing. Depending
```

_drag and drop_ or _rag & drop_ or _drag'n'drop_

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,8 +11,14 @@ Every entry has a category for which we use the following visual abbreviations:
...
+on which callback functions are supplied, they're activated, or not.
```
Comment (dominiklohmann) [inline CHANGELOG.md:18]:
```suggestion
on which callback functions are supplied they're activated.
```

**Review by dominiklohmann — APPROVED** [review]
Video suffices for me. I did not run this locally.

I glanced at the code and it looks alright to me. Just the usual changelog spelling.

**Comment by rolandpeelen** [comment]
> Video suffices for me. I did not run this locally.
> 
> I glanced at the code and it looks alright to me. Just the usual changelog spelling.

Thanks for the quick action 🚢

**Pull request merged**